### PR TITLE
depend on latest nipype master 2.0

### PR DESCRIPTION
**Pull request by shoshber**
replaces old pr

### CODE REVIEW

File: fmriprep/run_workflow.py [hunk 1]
```diff
@@ -16,11 +16,14 @@
 from argparse import ArgumentParser
 from argparse import RawTextHelpFormatter
 from lockfile import LockFile
+
 import logging
 from multiprocessing import cpu_count
 import os
 import os.path as op
 import glob
+import pkg_resources as pkgr
```
Comment (oesteban) [inline fmriprep/run_workflow.py]: pkgr is not used... why this import?

File: fmriprep/run_workflow.py [hunk 2]
```diff
@@ -128,6 +136,8 @@ def main():
         subject_list = [op.basename(subdir)[4:] for subdir in glob.glob(
             op.join(settings['bids_root'], 'sub-*'))]
 
+    logger.info("subject data is " + pprint.pformat(subject_list))
```
Comment (oesteban) [inline fmriprep/run_workflow.py]: Same as in the other PR, this is just a list so pprint is overkill the problem and adds one more import.

**Comment by oesteban** [comment]
any news on these two small comments?

File: fmriprep/run_workflow.py (hunk 1 above)
Comment (shoshber) [inline fmriprep/run_workflow.py]: cherrypicking is hard haha

**Pull request merged**